src/qml/RosterPage.qml

```diff
@@ -42,14 +44,13 @@ Kirigami.ScrollablePage {
 				});
 			}
 
-			// actions: [
-			// 	Kirigami.Action {
-			// 		iconName: "bookmark-remove"
-			// 		onTriggered: {
-			// 			//
-			// 		}
-			// 	}
-			// ]
+			actions: [
+			 	Kirigami.Action {
+			 		iconName: "bookmark-remove"
+			 		onTriggered: {
+					}
+				}
+			]
```

Comment:
Generally this is completely different thing and has nothing to do with Kg2, but it's ok.